libusb/libusbi.h

```diff
@@ -562,6 +563,9 @@ struct usbi_transfer {
 	uint32_t state_flags;   /* Protected by usbi_transfer->lock */
 	uint32_t timeout_flags; /* Protected by the flying_stransfers_lock */
 
+	/* This is used for logging mostly. As long as it is set, the */
```

Comment:
Is this missing the end of the sentence?

Reply:
Oh, that comment was from an earlier iteration of the patch.